content/news/buzz/january-2024-status-update/contents.lr

```diff
@@ -0,0 +1,58 @@
+title: January 2024 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+January has seen some long awaited and high profile features added to BeeWare.
+
+What we've done
+----------------
+
+* `PEP738 <https://peps.python.org/pep-0738/>`__, formalizing the introduction of Android as a Tier 3 platform in CPython, has been formally accepted as a draft. We're now revising the draft following community feedback, in anticipation of submitting the PEP for formal approval in the near future.
```

Comment:
I hadn't thought about it until recently but PEP references should include a space...presumably modeling something like RFCs. 

```suggestion
* `PEP 738 <https://peps.python.org/pep-0738/>`__, formalizing the introduction of Android as a Tier 3 platform in CPython, has been formally accepted as a draft. We're now revising the draft following community feedback, in anticipation of submitting the PEP for formal approval in the near future.
```